bitnami/grafana-operator/Chart.yaml

```
@@ -30,4 +30,4 @@ maintainers:
 name: grafana-operator
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/grafana-operator
-version: 4.4.15
+version: 4.4.16
```

Bump a minor version please
```suggestion
version: 4.5.0
```

---

Done